src/mono/wasm/debugger/DebuggerTestSuite/CustomViewTests.cs

```diff
@@ -64,5 +64,40 @@ public async Task UsingDebuggerTypeProxy()
                 ("listToTestToList.ToList()", TObject("System.Collections.Generic.List<int>", description: "Count = 11")));
 
         }
+
+        [Fact]
+        public async Task UsingDebuggerDisplayConcurrent()
+        {
+            async Task<bool> CheckProperties(JObject pause_location)
+            {
+                var locals = await GetProperties(pause_location["callFrames"][0]["callFrameId"].Value<string>());
+                var l = GetAndAssertObjectWithName(locals, "myList");
+                var val = l["value"];
+                if (val["description"].Value<string>() != "Count = 0")
+                    return false;
+                return true;
+            }
+
+            var bp = await SetBreakpointInMethod("debugger-test.dll", "DebuggerTests.DebuggerCustomViewTest2", "run", 2);
+            var pause_location = await EvaluateAndCheck(
+                "window.setTimeout(function() { invoke_static_method ('[debugger-test] DebuggerTests.DebuggerCustomViewTest2:run'); }, 1);",
+                "dotnet://debugger-test.dll/debugger-custom-view-test.cs",
+                bp.Value["locations"][0]["lineNumber"].Value<int>(),
+                bp.Value["locations"][0]["columnNumber"].Value<int>(),
+                "run");
+
+            pause_location = await StepAndCheck(StepKind.Over, "dotnet://debugger-test.dll/debugger-custom-view-test.cs", bp.Value["locations"][0]["lineNumber"].Value<int>()+2, bp.Value["locations"][0]["columnNumber"].Value<int>(),  "run");
+
+            System.Collections.Generic.List<Task<bool>> tasks = new System.Collections.Generic.List<Task<bool>>();
```

Comment:
Add a `using System.Collections.Generic;`, and change this to `List<Task<bool>> tasks = new();`

Reply:
done